cmake/dependencies/unix.cmake

```diff
@@ -1,4 +1,4 @@
 # unix specific dependencies
 # put anything here that applies to both linux and macos
 
-find_package(Boost COMPONENTS locale log filesystem program_options REQUIRED)
+find_package(Boost 1.75.0 COMPONENTS locale log filesystem program_options json REQUIRED)
```

Comment:
This is also going to require changes to CI workflows, docker files, and build instructions.

Also, not sure Boost 1.75.x is available on every distro yet. https://github.com/LizardByte/Sunshine/blob/nightly/docker/ubuntu-20.04.dockerfile#L40-L43

There was another PR that wanted to use boost json in the past but they ended up going a different route due to it not being available everywhere.